go/pkg/tool/tool.go

```diff
@@ -352,6 +353,39 @@ func (c *Client) DownloadAction(ctx context.Context, actionDigest, outputPath st
 	if _, err := c.GrpcClient.ReadProto(ctx, acDg, actionProto); err != nil {
 		return err
 	}
+
+	// Directory already exists, ask the user for confirmation before overwrite it.
+	if _, err := os.Stat(outputPath); !os.IsNotExist(err) {
+		fmt.Printf("Directory '%s' already exists. Do you want to overwrite it? (yes/no): ", outputPath)
+		var input string
+		if overwrite {
```

Comment:
You can avoid the `else` branch if you set `input="yes"` by default and then check `if !overwrite`.